rdrf/rdrf/services/io/defs/importer.py

```diff
@@ -515,6 +516,17 @@ def _create_registry_objects(self):
             config.consent_locked = registry_consent_locked
             config.save()
 
+        if "form_titles" in self.data:
```

Comment:
Shouldn't we validate that the `default_name`s from the yaml file match the ones we have defined in our model's choices?

Reply:
Updated.

Reply:
We will have to discuss this import logic a bit further.
I don't think the `order` should be in the filter at line `537`. We should ensure that the same validation that has been added to the AdminForm is also applied here.
Also, I'm not sure that we can't end up with inconsistent title customisations because we don't delete the existing ones but try to add each customisation from the YAML file.